cmd/relayer/setup/setup.go

```diff
@@ -270,65 +265,14 @@ func getPreRunInfo(home string) (*consts.RollappData, *consts.HubData, string, e
 
 	raData := consts.RollappData{
 		ID:     raID,
-		RpcUrl: fmt.Sprintf("%s:%d", raRpc, 443),
+		RpcUrl: raRpc,
 	}
 	return &raData, hd, kb, nil
 }
 
-func installRelayerDependencies(
-	home string,
-	raID string,
-	hd consts.HubData,
-) error {
-	raResp, err := rollapp.GetMetadataFromChain(raID, hd)
-	if err != nil {
-		return err
-	}
-
-	drsVersion, err := genesis.GetDrsVersionFromGenesis(home, raResp)
-	if err != nil {
-		pterm.Error.Println("failed to get drs version from genesis: ", err)
-		return err
-	}
-
-	drsInfo, err := firebaseutils.GetLatestDrsVersionCommit(drsVersion, hd.Environment)
-	if err != nil {
-		pterm.Error.Println("failed to retrieve latest DRS version: ", err)
-		return err
-	}
-
-	var raCommit string
-	switch strings.ToLower(raResp.Rollapp.VmType) {
-	case "evm":
-		raCommit = drsInfo.EvmCommit
-	case "wasm":
-		raCommit = drsInfo.WasmCommit
-	}
-
-	if raCommit == "UNRELEASED" {
-		return fmt.Errorf("rollapp does not support drs version: %s", drsVersion)
-	}
-
-	rbi := dependencies.NewRollappBinaryInfo(
-		raResp.Rollapp.GenesisInfo.Bech32Prefix,
-		raCommit,
-		strings.ToLower(raResp.Rollapp.VmType),
-	)
-
-	raDep := dependencies.DefaultRollappDependency(rbi)
-	err = dependencies.InstallBinaryFromRepo(raDep, raDep.DependencyName)
-	if err != nil {
-		return err
-	}
-
+func installRelayerDependencies() error {
```

Comment:
installation of rollapp binary is necessary because there are helper functions, like `LoadActiveChannel` which use the binary to query the rollapp for certain things. 

It is installed because there's an assumption that a relayer can be set up completely separately from the rollapp sequencer, where the binary isn't installed as part of `roller rollapp setup/init` commands.